packets2/willmsgresp_test.go

```diff
@@ -0,0 +1,62 @@
+package packets2
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	pkts "github.com/energomonitor/bisquitt/packets"
+)
+
+func TestWillMsgRespConstructor(t *testing.T) {
+	assert := assert.New(t)
+
+	pkt := NewWillMsgResp(RC_SUCCESS)
+
+	if pkt == nil {
+		t.Fatal("New packet should not be nil")
+	}
+
+	assert.Equal("*packets2.WillMsgResp", reflect.TypeOf(pkt).String(), "Type should be WillMsgResp")
+	assert.Equal(RC_SUCCESS, pkt.ReasonCode, "Default ReasonCode should be RC_ACCEPTED")
```

Comment:
The message mentions a different code than the one passed as the expected value.

Reply:
Fixed.